### PR TITLE
💾  Persist Wallet Connect sessions

### LINKED ISSUE #19843: 💾  Persist Wallet Connect requests
# Feature Issue

[comment]: # (Please replace ... with your information. Remove < and >)

### User Story

As a user, I want all the requests to be persisted, so that I can see my activity in the future.

### Description
We should use the persisted records for the "Connected dApps" list when offline, as using Wallet Conect's API might be unreliable when there's no connection. 

### Acceptance Criteria

- [ ] Connected dApps use persisted records to show the dApps when offline
- [x] Successful pairing requests are persisted

### Notes

Check with @stefandunca or @clauxx about the state of persistance on the status-go side.

### PR DESCRIPTION
**Pull request by shivekkhurana**
fixes #19843

### Summary

Building on top of Stefan's GO work, this PR saves every approved session to the backend. Also exposes methods to read sessions

#### Platforms
- Android
- iOS

### Steps to test
Skip Testing until feature flag is removed.

status: ready

### CODE REVIEW

File: src/status_im/contexts/wallet/wallet_connect/events.cljs [hunk 1]
```diff
@@ -185,5 +191,31 @@
...
-       {:fx [[:dispatch [:wallet-connect/pair scanned-text]]
-             [:dispatch [:dismiss-modal :screen/wallet.wallet-connect-session-proposal]]]}))))
+       {:fx [[:dispatch [:wallet-connect/pair scanned-text]]]}))))
+
+(rf/reg-event-fx
+ :wallet-connect/persist-session
+ (fn [_ [{:keys [id dapp-name dapp-url session-info]}]]
+   {:fx [[:json-rpc/call
+          [{:method     "wakuext_addWalletConnectSession"
+            :params     [{:id       (str id)
```
Comment (shivekkhurana) [inline src/status_im/contexts/wallet/wallet_connect/events.cljs:202]: Explicit `str` is needed, else status-go fails.

Go expects integer but id returned by WC is a string.

**Review by alwx — APPROVED** [review]

**Comment by shivekkhurana** [comment]
@qoqobolo Is this good to merge ?

**Comment by qoqobolo** [comment]
> @qoqobolo Is this good to merge ?

@shivekkhurana yep, go ahead 🟢

**Comment by jakubgs** [comment]
Please do not use emojis in commit titles. In PR titles it's fine, but commit titles are supposed to be plaintext. Just because GitHub renders them doesn't mean it's a good idea.

**Pull request merged**